### ISSUE TITLE
Gesture Label in MediaPipe Handpose Model

### ISSUE THREAD

**Issue by guillad** [kazuhitoyokoi/node-red-contrib-tensorflow#1]
REQUEST:
**********
Hi kazuhitoyokoi,
Awesome work! keep on the good stuff:-) 

As for theM ediaPipe Handpose Model, 
I believe that gesture-recognition, like in [here](https://1.bp.blogspot.com/-xmtORGq9oVQ/XVrTP5nvIDI/AAAAAAAAEh8/d7t_e2WVs5sSo94YsPxEfP3JlaBF1EDSwCLcBGAs/s320/image10.gif) and [here](https://1.bp.blogspot.com/-ym8EfnQKlqM/XVrTPqWUqsI/AAAAAAAAEh0/GfCY8BW28SgTqYO0gd230YgbwGgxc7yfgCLcBGAs/s320/image3.gif) is a MAJOR added value, to the existing X,Y coordinates. 

From this [blog](https://ai.googleblog.com/2019/08/on-device-real-time-hand-tracking-with.html#Gesture%20Recognition), it looks like the Gesture Recognizer is already implemented in TFlite. 
Is it Doable? Thanks!

![image](https://user-images.githubusercontent.com/57844677/83353985-f885e800-a35e-11ea-973d-8a22b3421cd8.png)

**Comment by kazuhitoyokoi** [comment]
@guillad I'm glad to hear your comment! This module is based on [Pre-trained TensorFlow.js models](https://github.com/tensorflow/tfjs-models/tree/master/handpose). Unfortunately, there is no gesture recognizer in the original repository. When the original repository supports the gesture recognizer, we will be able to support it in the handpose node soon. :)
